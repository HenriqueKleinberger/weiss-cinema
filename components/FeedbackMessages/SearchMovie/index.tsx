import { SEARCH_MOVIE } from '../../../constants/messages';
import MovieImg from '../../../public/logo.svg';
import { Container } from '../styles';

const SearchMovie = () => {
  return (
    <Container>
      <MovieImg width={100} height={100} />
      <p>{SEARCH_MOVIE}</p>
    </Container>
  );
};

export default SearchMovie;
