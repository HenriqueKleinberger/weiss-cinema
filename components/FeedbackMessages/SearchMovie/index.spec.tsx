import React from 'react';
import { shallow } from 'enzyme';
import SearchMovie from './index';

describe('SearchMovie', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<SearchMovie />);
    expect(wrapper).toMatchSnapshot();
  });
});
