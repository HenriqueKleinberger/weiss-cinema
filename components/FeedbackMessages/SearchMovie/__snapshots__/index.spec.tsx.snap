// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchMovie should match snapshot when is not  1`] = `
<styled.div>
  <logo.svg
    height={100}
    width={100}
  />
  <p>
    Search a movie.
  </p>
</styled.div>
`;
